collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/CascadingSelectTest.kt

```diff
@@ -0,0 +1,36 @@
+package org.odk.collect.android.feature.formentry
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain.chain
+
+@RunWith(AndroidJUnit4::class)
+class CascadingSelectTest {
+    private var rule = CollectTestRule()
+
+    @get:Rule
+    var copyFormChain: RuleChain = chain().around(rule)
+
+    @Test
+    fun cascadingSelect_withACSVFileWithColumnNamesStartingWithNumbers_shouldWorkCorrectly() {
```

Comment:
Do we have a reference to the original bug this was for? The case feels very specific, and I'm wondering if we can test it lower down.

Reply:
I can't find it.

Reply:
Let's add a comment that this was for a bug, but that the original issue is missing.

Reply:
Done.